lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/cc/QtiImport.java

```diff
@@ -1078,7 +1101,11 @@ boolean procshort(Node item) throws IOException{
 	out.println("    <qtimetadata>");
 	out.println("      <qtimetadatafield>");
 	out.println("        <fieldlabel>qmd_itemtype</fieldlabel>");
-	out.println("        <fieldentry>Fill In the Blank</fieldentry>");
+	if (type.equalsIgnoreCase("Fill In the Blank")) {
```

Comment:
IMO, "Fill In the Blank" should be defined in a const somewhere. This string is all over the place in the Samigo and Lessons code and you would be doing the community a service by cleaning those up.

Reply:
@adrianfish, I'm agree with you but these 2 classes (QtiImport.java and SamigoExport.java) are really a big disaster about: no const, no spacing, no braces, bad formatting, unnecessary comments etc.
IMO all that things should be resolved on a new one. Please, file a new jira for that purpose. Thanks!